logging/config/cluster-logging-curator.adoc

```diff
@@ -5,14 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} {product-version}, log curation is performed by Elasticsearch Curator, based on a xref:../../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[retention policy] that you define.
+In {product-title} {product-version}, log curation is performed by ElasticSearch `IndexManagement` - the `elasticsearch-im-app`, `elasticsearch-im-audit` and `elasticsearch-im-infra` cronjobs, which you can configure using the xref:../../logging/config/cluster-logging-log-store.html#cluster-logging-elasticsearch-retention_cluster-logging-store[retentionPolicy].
 
-The Elasticsearch Curator tool removes Elasticsearch indices that use the data model prior to {product-title} 4.5. You can modify the Curator index retention policy for your old data.
+Elasticsearch Curator will be deprecated in a future version. If the cluster was deployed on {product-title} version 4.5 or later, Elasticsearch Curator is not required.
 
-// The following include statements pull in the module files that comprise
-// the assembly. Include any combination of concept, procedure, or reference
-// modules required to cover the user story. You can also include other
-// assemblies.
+The Elasticsearch Curator tool removes Elasticsearch indices that use the data model prior to {product-title} 4.5. To modify the Curator index retention policy for your old data, edit the `ClusterLogging` custom resource as described here.
```

Comment:
I'm not certain how to best convey but we discovered the new index management jobs will actually remove indices from the old model and new meaning curator isnt really required at all

Reply:
Updated to reflect this information.